packages/libs/coreui/src/components/containers/DraggablePanel/index.tsx

```diff
@@ -1,28 +1,28 @@
-import { CSSProperties, ReactNode, useEffect, useState } from "react";
```

Comment:
🤔 I wonder if my editor config is fighting with some other config...

Reply:
Ignore this file for now, I suppose.

Reply:
After every commit the code should be going through `prettier` so our editors' quirks shouldn't matter. However, during the monorepo migration, something happened in this arena...

Reply:
> I think CoreUI had a different config

Ahh, that makes total sense! We could do a massive formatting pr in the monorepo!